### PR TITLE
Alex likes

### LINKED ISSUE #3: Likes CRUD
- Create action already working
- Destroy action needed
- Update action not needed

### PR DESCRIPTION
**Pull request by Alex-Desjardins**

### CODE REVIEW

File: app/controllers/graphql_controller.rb [hunk 1]
```diff
@@ -21,30 +21,28 @@ def execute
 
   private
 
-  # Handle variables in form data, JSON body, or a blank value
-  def prepare_variables(variables_param)
-    case variables_param
+  # Handle form data, JSON body, or a blank value
+  def ensure_hash(ambiguous_param)
```
Comment (ggerdsen) [inline app/controllers/graphql_controller.rb:25]: You'll have to get me up to speed on this.

Comment (Alex-Desjardins) [reply app/controllers/graphql_controller.rb:25]: I have no clue what this is! This change must've happened under the hood

File: app/graphql/types/mutation_type.rb [hunk 2]
```diff
@@ -1,10 +1,7 @@
...
   class MutationType < Types::BaseObject
-    # TODO: remove me
-    field :test_field, String, null: false,
-      description: "An example field added by the generator"
-    def test_field
-      "Hello World"
-    end
+    field :create_user, mutation: Mutations::CreateUser
+    field :create_like, mutation: Mutations::CreateLike
+    field :destroy_like, mutation: Mutations::DestroyLike
```
Comment (ggerdsen) [inline app/graphql/types/mutation_type.rb:5]: Looks like we have some mutations missing.

Comment (Alex-Desjardins) [reply app/graphql/types/mutation_type.rb:5]: Yea this was branched before final User and Post branches were merged in to main.

**Review by ggerdsen — COMMENTED** [review]
I'm not sure this is the right branch to merge from?  Seems like it has some missing mutations in mutation_type.rb?

File: app/controllers/graphql_controller.rb (hunk 1 above)
Comment (ggerdsen) [reply app/controllers/graphql_controller.rb:25]: I thought you'd whipped up some "rails magic" for us! hahaha

**Comment by Alex-Desjardins** [comment]
> I'm not sure this is the right branch to merge from? Seems like it has some missing mutations in mutation_type.rb?

There are tons of merge conflicts with this branch so we will make sure to keep everything thats on main branch regarding gems etc.

**Pull request merged**